Libraries/Components/CheckBox/AndroidCheckBoxNativeComponent.js

```diff
@@ -1,74 +0,0 @@
-/**
```

Comment:
Seems like we can remove https://github.com/microsoft/react-native-macos/blob/master/Libraries/Components/CheckBox/CheckBox.macos.js

Reply:
And any Mac specific VM code for it if there was any